Resources/Prototypes/_Goobstation/Roles/Jobs/Dignitary/blueshield_officer.yml

```
@@ -51,20 +54,28 @@
   supervisors: job-supervisors-ntr-centcom
   canBeAntag: false
   access:
-  - Security
+  - Atmospherics
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)